src/plugins/tools/itemThemeSwitcher/itemThemeSwitcher.js

```diff
@@ -260,8 +260,10 @@ export default pluginFactory({
      * Called during the runner's destroy phase
      */
     destroy: function destroy() {
-        this.storage.setItem('itemThemeId', '');
         shortcut.remove(`.${this.getName()}`);
+        return this.getTestRunner().getPluginStore(this.getName()).then(function(itemThemesStore) {
+            return itemThemesStore.setItem('itemThemeId', '');
```

Comment:
Is it possible to use `removeItem` method?

Reply:
Yes, thanks :)